esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/site/metoffice.cylc

```diff
@@ -41,6 +41,12 @@
         [[[directives]]]
             --mem = 3G
 
+    [[process<fast=recipe_ocean_multimap>]]
+        # Actual: 6m52.260589s, 2.5 GB on 2024-05-28.
```

Comment:
Rounding to the nearest second here would be fine 😊

Reply:
Done here: 56be2739d